packages/common/src/test/setupTests.ts

```diff
@@ -0,0 +1,34 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
+
+type LocalStorageMockProps = {
+  store: any;
+};
+
+class LocalStorageMock {
+  store: any;
+  constructor() {
+    this.store = {};
+  }
+
+  clear() {
+    this.store = {};
+  }
+
+  getItem(key: string) {
+    return this.store[key] || null;
+  }
+
+  setItem(key: string, value: string) {
+    this.store[key] = value.toString();
+  }
+
+  removeItem(key: string) {
+    delete this.store[key];
```

Comment:
Maybe reassign instead of using `delete` so we don't mutate the `store` directly? 🤔

```ts
const { [key]: deletedKey, ...storeRest } = store;

this.store = storeRest;
```